src/engine/sidechain/enginebroadcast.h

```diff
@@ -25,6 +25,13 @@ class EngineBroadcast
         : public QThread, public NetworkStreamWorker {
     Q_OBJECT
   public:
+	enum StatusCOStates {
+	    STATUSCO_UNCONNECTED = 0, // IDLE state, no error
+	    STATUSCO_CONNECTING = 1, // 30 s max
+	    STATUSCO_CONNECTED = 2, // On Air
+		STATUSCO_FAILURE = 3 // Happens when disconnected by an error
```

Comment:
be sure to set eclipse to spaces only.
https://www.mixxx.org/wiki/doku.php/eclipse#eclipse_code_formatter